modules/negotiate/lib/Auth/Source/Negotiate.php

```diff
@@ -136,6 +138,12 @@ public function authenticate(&$state)
                     SimpleSAML\Logger::debug('Negotiate - authenticate(): No "Negotiate" found. Skipping.');
                 }
             }
+            
+            if ($this->rcache === true) {
+                putenv("KRB5RCACHETYPE=dfl");
+            } elseif ($this->rcache === false) {
```

Comment:
The default would in this case be the system default...
If you don't set the option, the system default applies. If you do set it, your setting should override the system default, so that's why I did it this way.
I must admit I have not enough knowledge about Kerberos.. I have no clue about other possible values.